src/modules/85-cv/components/ExecutionVerification/components/LogAnalysisContainer/components/LogAnalysisRow/components/LogAnalysisDataRow/components/LogAnalysisRowMetadata.tsx

```diff
@@ -0,0 +1,79 @@
+import { Button, ButtonVariation, Container, Layout, Text } from '@harness/uicore'
+import React from 'react'
+import type { LogFeedback, TicketResponseDto } from 'services/cv'
+import { useStrings } from 'framework/strings'
+import RiskItemIndicator from '../../UpdateEventPreferenceDrawer/component/component/RiskItemIndicator'
+import LogAnalysisRiskDisplayTooltip from './components/LogAnalysisRiskDisplayTooltip'
+import css from '../../../LogAnalysisRow.module.scss'
+
+interface LogAnalysisRowMetadataProps {
+  isFeedbackTicketPresent: boolean
+  isFeedbackApplied: boolean
+  feedbackApplied?: LogFeedback
+  ticketDetails?: TicketResponseDto
+}
+
+export const LogAnalysisRowMetadata = ({
+  feedbackApplied,
+  ticketDetails,
+  isFeedbackApplied,
+  isFeedbackTicketPresent
+}: LogAnalysisRowMetadataProps): JSX.Element | null => {
+  const { getString } = useStrings()
+
+  const openJiraTicket = (): void => {
+    window.open(ticketDetails?.url, '_blank')?.focus()
+  }
+
+  if (!isFeedbackApplied && !isFeedbackTicketPresent) {
+    return null
+  }
+
+  const getFeedbackContent = (): JSX.Element | null => {
+    if (isFeedbackApplied) {
+      return (
+        <Container data-testid="feedbackContainer" className={css.rowAppliedFeedbackIndicator}>
+          <RiskItemIndicator isSmall risk={feedbackApplied?.feedbackScore} />
+          <Text
+            tooltip={<LogAnalysisRiskDisplayTooltip feedback={feedbackApplied} />}
+            tooltipProps={{ isDark: true }}
+            className={css.rowAppliedFeedbackIndicatorText}
+          >
+            {getString('cv.logs.feedbackApplied')}
+          </Text>
+        </Container>
+      )
+    }
+
+    return null
+  }
+
+  const getTicketContent = (): JSX.Element | null => {
+    if (isFeedbackTicketPresent) {
+      return (
+        <Button
+          className={css.jiraTicketLink}
+          data-testid="createdJiraTicketIdDisplay"
+          onClick={openJiraTicket}
+          variation={ButtonVariation.LINK}
+          icon="service-jira"
+          withoutCurrentColor
+          iconProps={{ size: 16 }}
+        >
+          {ticketDetails?.externalId}
+        </Button>
+      )
+    }
+
+    return null
+  }
+
+  return (
+    <Container padding={{ left: 'medium', right: 'medium' }}>
+      <Layout.Horizontal className={css.logsMetadataHolder}>
```

Comment:
we can convert getFeedbackContent and getTicketContent to components 
```suggestion
<Layout.Horizontal className={css.logsMetadataHolder}>
       { isFeedbackApplied && <Feedback />}
       { isFeedbackTicketPresent &&  <Ticket />}
</Layout.Horizontal>
```